### PR TITLE
Modify GeneratorModification functional logs

### PR DESCRIPTION
**Pull request by ayolab**
Signed-off-by: Ayoub LABIDI <ayoub.labidi@protonmail.com>

### CODE REVIEW

File: src/main/java/org/gridsuite/modification/server/service/NetworkModificationService.java [hunk 1]
```diff
@@ -750,49 +781,55 @@ private void modifyGeneratorMinMaxReactiveLimits(GeneratorModificationInfos modi
             }
         }
 
-        // (if the min and max reactive limits are null and there is no previous min max limits set we set them to Double max and
+        // (if the min and max reactive limits are null and there is no previous min max
+        // limits set we set them to Double max and
         // Double min values)
```
Comment (laurgarn) [inline src/main/java/org/gridsuite/modification/server/service/NetworkModificationService.java:786]: While you were re-wrapping, you could lift up "Double min values" to the line above

File: src/main/java/org/gridsuite/modification/server/service/NetworkModificationService.java [hunk 2]
```diff
@@ -750,49 +781,55 @@ private void modifyGeneratorMinMaxReactiveLimits(GeneratorModificationInfos modi
...
-        // The user can change the value of MinimumReactivePower, MaximumReactivePower or both
+        // The user can change the value of MinimumReactivePower, MaximumReactivePower
```
Comment (laurgarn) [inline src/main/java/org/gridsuite/modification/server/service/NetworkModificationService.java:787]: Is it necessary to re-wrap comments ?

Comment (ayolab) [reply src/main/java/org/gridsuite/modification/server/service/NetworkModificationService.java]: it's due to IDE formatting, done

File: src/main/java/org/gridsuite/modification/server/service/NetworkModificationService.java (hunk 1 above)
Comment (ayolab) [reply src/main/java/org/gridsuite/modification/server/service/NetworkModificationService.java]: Done

**Review by EstherDarkish — APPROVED** [review]

**Review by laurgarn — APPROVED** [review]
Shallow OK

**Pull request merged**